backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/model/CreditStatusEnum.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2021 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.bpartner.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public enum CreditStatusEnum
```

Comment:
name it misleading.
rename it as SetCreditStatusEnum because it's not THE CreditStatus.

move it near the process because besides that process nowhere is needed / useful.

Implement `ReferenceListAwareEnum` and required static methods.
See an example here org.eevolution.api.BOMType.

Reply:
rename the enum to SetCreditStatusEnum and implements ReferenceListAwareEnum 
moved it near the process